omnibus/config/software/datadog-windows-procmon-driver.rb

```diff
@@ -0,0 +1,21 @@
+name "datadog-windows-procmon-driver"
```

Comment:
Let's add the notice header here:
```
# Unless explicitly stated otherwise all files in this repository are licensed
# under the Apache License Version 2.0.
# This product includes software developed at Datadog (https:#www.datadoghq.com/).
# Copyright 2016-present Datadog, Inc.
```

Reply:
Added, thanks for noticing this!